src/pages/AccountPage.tsx

```diff
@@ -0,0 +1,82 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import { Container, Card, Badge } from 'react-bootstrap'
+import { useNavigate } from 'react-router-dom'
+import useTranslationHook from '../hook/TranslationHook'
+
+const AccountPage = (): JSX.Element => {
+  const navigate = useNavigate()
+  const { t } = useTranslationHook()
+
+  const goToProfile = (): void => {
+    navigate('account/profile')
+  }
+  return (
+    <>
+      <Container>
+        {/* ToDo add profil-header__avatar */}
+        <div className='profil-header__body'>
+          <h1 className='h1'>{t('title.myAccount')}</h1>
+          <p className='hstack'>
+            <Badge bg='secondary'>{t('text.standardAccount')}</Badge>
+          </p>
+        </div>
```

Comment:
Do you really want an avatar, why would we want that?

Reply:
Not necessarily